<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Leaderboard</class>
 <widget class="QWidget" name="Leaderboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>1000</width>
     <height>1030</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPassedLevels">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #4CAF50;
    color: black;
    border: none;
    padding: 10px;
    border-radius: 5px;
}
QPushButton:hover {
    background-color: #45a049;
}</string>
   </property>
   <property name="text">
    <string>通关关卡数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBestTimes">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #2196F3;
    color: black;
    border: none;
    padding: 10px;
    border-radius: 5px;
}
QPushButton:hover {
    background-color: #0b7dda;
}</string>
   </property>
   <property name="text">
    <string>关卡最佳时间</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMedals">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ff9800;
    color: black;
    border: none;
    padding: 10px;
    border-radius: 5px;
}
QPushButton:hover {
    background-color: #e68a00;
}</string>
   </property>
   <property name="text">
    <string>勋章获得数</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
